internal/indexer/db/client/finality_provider.go

```
@@ -83,7 +80,7 @@ func (indexerdbclient *IndexerDatabase) GetFinalityProviders(
 		client,
 		filter,
 		opts,
-		finalityProvidersLimit,
+		indexerdbclient.Cfg.MaxPaginationLimit,
```

please add validation of this value (if it's empty it should throw an error, otherwise we will forget about it when it's time to deploy) and update config/*.yml files if they don't have this limit set

---

@kirugan validation already exists in `internal/shared/config/db.go`:
```go
if cfg.MaxPaginationLimit < 2 {
        return fmt.Errorf("max pagination limit must be greater than 1")
}
```

And config files `config-local.yml` and `config-docker.yml` already have the `max-pagination-limit` field set